sdk/boms/azure-sdk-bom/pom.xml

```diff
@@ -47,322 +44,259 @@
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-storage-file-share</artifactId>
-        <version>12.9.1</version>
+        <version>12.10.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-storage-internal-avro</artifactId>
-        <version>12.0.4</version>
+        <version>12.0.5</version>
       </dependency>
-
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-storage-queue</artifactId>
-        <version>12.9.1</version>
+        <version>12.10.0</version>
       </dependency>
-
-      <!-- Dependencies used by the Azure SDKs -->
       <dependency>
         <groupId>com.fasterxml.jackson</groupId>
         <artifactId>jackson-bom</artifactId>
-        <version>2.12.2</version>
+        <version>2.12.3</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-
+      <dependency>
+        <groupId>com.fasterxml.jackson.module</groupId>
+        <artifactId>jackson-module-afterburner</artifactId>
+        <version>2.12.2</version>
+      </dependency>
       <dependency>
         <groupId>com.google.code.gson</groupId>
         <artifactId>gson</artifactId>
         <version>2.8.6</version>
       </dependency>
-
       <dependency>
         <groupId>com.microsoft.azure</groupId>
         <artifactId>msal4j</artifactId>
-        <version>1.10.1</version>
+        <version>1.10.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.microsoft.azure</groupId>
         <artifactId>msal4j-persistence-extension</artifactId>
         <version>1.1.0</version>
       </dependency>
-
       <dependency>
         <groupId>com.microsoft.azure</groupId>
         <artifactId>qpid-proton-j-extensions</artifactId>
-        <version>1.2.3</version>
-      </dependency>
-
-      <dependency>
-        <groupId>com.nimbusds</groupId>
-        <artifactId>oauth2-oidc-sdk</artifactId>
-        <version>9.4</version>
+        <version>1.2.4</version>
       </dependency>
-
       <dependency>
         <groupId>com.squareup.okhttp3</groupId>
         <artifactId>okhttp</artifactId>
         <version>4.8.1</version>
       </dependency>
-
       <dependency>
         <groupId>io.dropwizard.metrics</groupId>
         <artifactId>metrics-core</artifactId>
-        <version>4.1.19</version>
+        <version>4.1.0</version>
       </dependency>
-
       <dependency>
         <groupId>io.micrometer</groupId>
         <artifactId>micrometer-core</artifactId>
-        <version>1.6.6</version>
+        <version>1.7.0</version>
       </dependency>
-
       <dependency>
         <groupId>io.netty</groupId>
         <artifactId>netty-bom</artifactId>
-        <version>4.1.63.Final</version>
+        <version>4.1.65.Final</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-
-      <dependency>
-        <groupId>io.netty</groupId>
```

Comment:
Any reason on why this was removed?

Reply:
I am not sure why this was there in the first place. The one below it comes because azure-core takes direct dependency on it but none of our libraries directly include this AFAIK.